src/content/docs/en/guides/cms/kontent-ai.mdx

```diff
@@ -0,0 +1,548 @@
+
+### Prerequisites
+
+To get started, you'll need the following:
+
+1. **Kontent.ai project** - If you don't have a Kontent.ai account yet, [sign up for free](https://app.kontent.ai/sign-up) and create a new project.
+
+2. **Delivery API keys** - You will need the Environment ID for published content and the Preview API key for fetching drafts (optional). Both keys are located in the **Environment Settings -> API keys** tab in Kontent.ai.
+
+### Setting up credentials
+
+To add your Kontent.ai credentials to Astro, create a `.env` file in the root of your project with following variables:
+
+```ini title=".env"
+KONTENT_ENVIRONMENT_ID=YOUR_ENVIRONMENT_ID
+KONTENT_PREVIEW_API_KEY=YOUR_PREVIEW_API_KEY
+```
+
+Now, these environment variables can be used in your Astro project.
+
+If you would like to get [TypeScript IntelliSense for these environment variables](/en/guides/environment-variables/#intellisense-for-typescript), you can create a new `env.d.ts` file in the `src/` directory and configure `ImportMetaEnv` like this:
+```ts title="src/env.d.ts"
+interface ImportMetaEnv {
+  readonly KONTENT_ENVIRONMENT_ID: string;
+  readonly KONTENT_PREVIEW_API_KEY: string;
+}
+```
+
+Your root directory should now include these new files:
+
+<FileTree title="Project Structure">
+- src/
+  - **env.d.ts**
+- **.env**
+- astro.config.mjs
+- package.json
+</FileTree>
+
+
+### Installing dependencies
+
+To connect Astro with your Kontent.ai project, install the [Kontent.ai TypeScript SDK](https://github.com/kontent-ai/delivery-sdk-js):
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+    npm install @kontent-ai/delivery-sdk
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+    pnpm install @kontent-ai/delivery-sdk
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+    yarn add @kontent-ai/delivery-sdk
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Next, create a new file called `kontent.ts` in the `src/lib/` directory of your Astro project.
+
+```ts title="src/lib/kontent.ts"
+import { createDeliveryClient } from "@kontent-ai/delivery-sdk";
+
+export const deliveryClient = createDeliveryClient({
+    environmentId: import.meta.env.KONTENT_ENVIRONMENT_ID,
+    previewApiKey: import.meta.env.KONTENT_PREVIEW_API_KEY,
+});
+```
+
+:::note[Note]
+Read more on [getting environment variables in Astro.](https://docs.astro.build/en/guides/environment-variables/#getting-environment-variables).
+:::
+
+This implementation creates a new `DeliveryClient` object using credentials from the `.env` file.
+
+:::note[previews]
+The `previewApiKey` is optional. When used, you can [configure each query](https://github.com/kontent-ai/delivery-sdk-js#enable-preview-mode-per-query) to the Delivery API endpoint to return the latest versions of content items regardless of their state in workflow. Otherwise, only published items are returned.*
```

Comment:
Missing "the" and  accidental `*`
```suggestion
The `previewApiKey` is optional. When used, you can [configure each query](https://github.com/kontent-ai/delivery-sdk-js#enable-preview-mode-per-query) to the Delivery API endpoint to return the latest versions of content items regardless of their state in the workflow. Otherwise, only published items are returned.
```